src/main/scala/edu/ie3/util/scala/quantities/ApparentPower.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.util.scala.quantities
+
+import squants._
+import squants.energy._
+import squants.time.Time
+
+import scala.math.{acos, sin}
+import scala.util.Try
+
+final class ApparentPower private (
+    val value: Double,
+    val unit: ApparentPowerUnit,
+) extends Quantity[ApparentPower] {
+
+  def dimension: ApparentPower.type = ApparentPower
+
+  def toMillivoltamperes: Double = to(Millivoltamperes)
+  def toVoltamperes: Double = to(Voltamperes)
+  def toKilovoltamperes: Double = to(Kilovoltamperes)
+  def toMegavoltamperes: Double = to(Megavoltamperes)
+  def toGigavoltamperes: Double = to(Gigavoltamperes)
+
+  def toPower(cosPhi: Double): Power = Watts(toVoltamperes * cosPhi)
+
+  def toReactivePower(cosPhi: Double): ReactivePower =
+    Vars(toVoltamperes * sin(acos(cosPhi)))
```

Comment:
S multiplied sin(cosPhi) should be reactive power, or?

Reply:
Ok found my failure. You're right. Not sure if there is a better solution for writing this. Maybe some scala doc for this class would be a good idea?